src/components/RequestARepModalForm.astro

```diff
@@ -302,6 +302,10 @@
     </div>
   </div>
 </div>
+<style>
```

Comment:
Instead of this, do the import in the Layout at the end of the day, we need it on every single plage, you could do the same for the script.

Reply:
Something like:

```
import "/node_modules/nice-select2/src/scss/nice-select2.scss";
```